src/mjolnir/graphbuilder.cc

```diff
@@ -120,7 +120,7 @@ void ConstructEdges(const OSMData& osmdata, const std::string& ways_file,
           const std::string& nodes_file,
           const std::string& edges_file, const float tilesize,
           const std::function<GraphId (const OSMNode&)>& graph_id_predicate) {
-  LOG_INFO("Creating graph edges from ways...")
+  LOG_INFO("Creating graph edges from ways...");
```

Comment:
lots of these. i thought it was dumb that the macro included a semicolon because this would force you to not be allowed to use a log line in a single line if (because it would be a double semicolon)